wiki/libraries/loom/markdown/loom.md

```diff
@@ -0,0 +1 @@
+# Loom
```

Comment:
I would add some content to this page explaining loom and linking to the different pages in the explaination